timeless/sms.py

```diff
@@ -13,6 +15,28 @@ def send(self):
         raise NotImplementedError
 
 
+class RetrySendSMS:
+    """Decorator class for resending requests"""
+    def __init__(self, retry_count=5, timeout=1):
+        self.retry_count = retry_count
+        self.timeout = timeout
+        self.counter = 0
+
+    def __call__(self, fn):
+        @functools.wraps(fn)
```

Comment:
@vladarefiev what is ``fn``?

Reply:
@amihaiemil function. it's usual abbreviation in python decorators. but what do you prefer?

Reply:
@vladarefiev I understood the decorator's idea now, thanks! Could we name it to ``send``? Since that's the method that is actually called.

Reply:
@amihaiemil yes, will do it